src/meeting/speakersObserver.ts

```
@@ -0,0 +1,115 @@
+import { Page } from '@playwright/test'
+import { MeetingProvider, RecordingMode, SpeakerData } from '../types'
+import { MeetSpeakersObserver } from './meet/speakersObserver'
+import { TeamsSpeakersObserver } from './teams/speakersObserver'
+
+export class SpeakersObserver {
+    private meetingProvider: MeetingProvider
+    private observer: MeetSpeakersObserver | TeamsSpeakersObserver | null = null
+    private isObserving: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)